src/main/resources/templates/fragments/icons.html

```diff
@@ -1967,6 +1967,84 @@ <h1>Icons</h1>
         </svg>
       </td>
     </tr>
+    <tr>
+      <td>submitAdditionalIcons</td>
+      <td>
+        <svg th:fragment="submitAdditionalIcons" aria-hidden="true" width="35" height="35"
+             viewBox="0 0 35 35" fill="none" xmlns="http://www.w3.org/2000/svg">
+          <g clip-path="url(#clip0_114_502)">
+            <path
+                d="M20.4167 2.91699H8.75001C7.14584 2.91699 5.84793 4.22949 5.84793 5.83366L5.83334 29.167C5.83334 30.7712 7.13126 32.0837 8.73543 32.0837H26.25C27.8542 32.0837 29.1667 30.7712 29.1667 29.167V11.667L20.4167 2.91699ZM23.3333 26.2503H11.6667V23.3337H23.3333V26.2503ZM23.3333 20.417H11.6667V17.5003H23.3333V20.417ZM18.9583 13.1253V5.10449L26.9792 13.1253H18.9583Z"
+                fill="black"/>
+          </g>
+          <defs>
+            <clipPath id="clip0_114_502">
+              <rect width="35" height="35" fill="white"/>
+            </clipPath>
+          </defs>
+        </svg>
+      </td>
+    </tr>
+    <tr>
+      <td>downloadDocumentsIcon</td>
+      <td>
+        <svg th:fragment="downloadDocumentsIcon" aria-hidden="true" width="30" height="35"
+             viewBox="0 0 30 35" fill="none" xmlns="http://www.w3.org/2000/svg">
+          <path
+              d="M26 12.6471L18.5714 12.6471V1L7.42857 1L7.42857 12.6471L0 12.6471L13 26.2353L26 12.6471ZM0 30.1177L0 34H26V30.1177L0 30.1177Z"
+              fill="black"/>
+        </svg>
+      </td>
+    </tr>
+    <tr>
+      <td>successComputerPrimaryIcon</td>
```

Comment:
We haven't usually put the word "Icon" in the name of the item, just the description of what it is.  Is it just a computer?

Reply:
Maybe this could be  `computerWithCheckmark`?

Reply:
Actually, I take this back.  We have a varied practice now with the word "Icon", so what you did is not inconsistent with what's in the file, but I think we should take the word Icon out from all spots.

Reply:
Updated to: computerWithCheckmark